Podfile.lock

```diff
@@ -60,12 +57,11 @@ SPEC CHECKSUMS:
   GTMSessionFetcher: e8647203b65cee28c5f73d0f473d096653945e72
   JWTDecode: 33e5e26e5ddbd21b2065820894c08fef0cad9509
   LBBottomSheet: 8ee6d3a86fcfc583bec73aa3bfcb776b2763f59d
-  lottie-ios: 8f97d3271e155c2d688875c29cd3c74908aef5f8
+  lottie-ios: 809ecf2d460ed650a6aed7aa88b2ec45fab4779c
   ReSwift: 929afd2c897f0d33c76c730710a31bb2d2a66713
   ReSwiftThunk: 7ca5145764be6f30b62db88a52d170133954629e
-  Sodium: 23d11554ecd556196d313cf6130d406dfe7ac6da
   SwiftKeychainWrapper: 807ba1d63c33a7d0613288512399cd1eda1e470c
 
-PODFILE CHECKSUM: 981c94972605c358aa2f64d87871cd61ff862708
+PODFILE CHECKSUM: 03d1c5cfa3bb202ff40b08aece5e5e0c0f733dd0
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.11.2
```

Comment:
downgrade?

Reply:
Not sure what's happening there. I'm actually thinking we could just add the `Podfile.spec` file to `.gitignore`. I'm not sure if it's required to build packages for SPM or cocoaPods

Reply:
FIle has been removed